### ISSUE TITLE
hasFiles() does not support arrays

### ISSUE THREAD

**Issue by Livijn** [sahibalejandro/form-object#6]
Images are not uploaded/parsed when using an array. The `hasFiles()` doesn't do a deep search.

Example of structure:
```
data: {
    name: 'oh hi mark',
    images: [ File, File ],
}
```

Potential solution:
* Allow a `bool forceFormDataObject` in the `Form()` constructor.

**Comment by sahibalejandro** [comment]
The solution for this is to implement array for all fields, not just images.
Working on.

**Comment by sahibalejandro** [comment]
Fixed on v1.7.0 https://github.com/sahibalejandro/form-object/releases/tag/v1.7.0
